packages/atomic/cypress/integration/facet-date.cypress.ts

```diff
@@ -0,0 +1,233 @@
+  });
+
+  describe('When page is loaded', () => {
+    it('Numeric facet should load, pass accessibility test and have correct label', () => {
+      validateFacetComponentLoaded(
+        dateFacetProp.label,
+        FacetSelectors.facetDate
+      );
+    });
+
+    it('Should generate all custom ranges', () => {
+      cy.getTextOfAllElements('@allFacetValueLabel').then((elements) => {
+        Object.keys(dateRange).forEach((i) => {
+          const facetValueConverted = convertDateToFacetValue(
+            (dateRange as any)[i]
+          );
+          expect(elements).to.include(facetValueConverted);
+        });
+      });
+    });
+
+    it('Should generate correct number of custom ranges', () => {
+      const totalCustomRangeLength = Object.keys(dateRange).length;
+      validateFacetNumberofValueEqual(totalCustomRangeLength);
+    });
+
+    describe.skip('When select 1 facetValue checkbox', () => {
+      it('Should activate checkbox and log UA', () => {
+        assertBasicFacetFunctionality('@firstFacetValue', dateFacetProp.field);
+      });
+    });
+  });
+});
+
+describe('Date facet contains range returns 0 result', () => {
+  it('Should hide the range automatically', () => {
+    setUpPage(`
+    <atomic-date-facet field="${dateFacetProp.field}" label="${dateFacetProp.label}">
+        <atomic-date-range start='01/01/2006' end='01/01/2014' end-inclusive="true"></atomic-date-range>
+        <atomic-date-range start='01/01/2014' end='01/01/2021' end-inclusive="true"></atomic-date-range>
+        <atomic-date-range start='01/01/20014' end='01/01/20021' end-inclusive="true"></atomic-date-range>
+    </atomic-date-facet>`);
+    createAliasFacetUL(dateFacetProp.field, FacetSelectors.facetDate);
+    assertNonZeroFacetCount();
+  });
+});
+
+describe('Date with custom date-format', () => {
+  it('Should render correct date format', () => {
+    setUpPage(`
+    <atomic-date-facet field="${dateFacetProp.field}" label="${dateFacetProp.label}" date-format="DD/MMM/YYYY">
+          <atomic-date-range start='01/01/2006' end='01/01/2014' end-inclusive="true"></atomic-date-range>
+          <atomic-date-range start='01/01/2014' end='01/01/2021' end-inclusive="true"></atomic-date-range>
+    </atomic-date-facet>`);
+    createAliasFacetUL(dateFacetProp.field, FacetSelectors.facetDate);
+    validateFacetComponentLoaded(dateFacetProp.label, FacetSelectors.facetDate);
+
+    const formatedStart = convertDateFormatLabel('01/01/2006', 'DD/MMM/YYYY');
+    const formatedEnd = convertDateFormatLabel('01/01/2014', 'DD/MMM/YYYY');
+    const formatedLabel = `${formatedStart} to ${formatedEnd}`;
+
+    cy.get('@firstFacetValue')
+      .find('label span:nth-child(1)')
+      .should('contain.text', formatedLabel);
+  });
+});
+
+describe('Date with invalid options', () => {
+  describe('When date facet uses invalid field/field returns no result', () => {
+    it('Should hide the facet', () => {
```

Comment:
Is the describe necessary is there is just one test? Can the description be moved in the "it"
Or perhaps I've been writing test wrong and context is always in a Describe 🤔